### PR TITLE
Immutable built in test material #1446

### PR DESCRIPTION
**Pull request by kz0817**
These patches makes built-in test materials immutable in order to
prevent them from being changed by mistake.

### CODE REVIEW

**Comment by cosmo0920** [comment]
It seems to be good.

But I have a question.
C++11 reference uses following style range based for:

```cpp
attr(optional) for ( range_declaration : range_expression ) loop_statement		
``` 

Why do you use `range_declaration: range_expression` style?

**Comment by kz0817** [comment]
> Why do you use range_declaration: range_expression style?

I wasn't aware that.
What is 'C++11 reference' you mentioned ?
Could you please tell me the pointer to that ?

**Comment by cosmo0920** [comment]
Oh, sorry.

This is cpp reference: http://en.cppreference.com/w/cpp/language/range-for

**Comment by kz0817** [comment]
> Here is cpp reference: http://en.cppreference.com/w/cpp/language/range-for#Syntax

OK. I'll fix to follow the style.

**Comment by kz0817** [comment]
I fixed. Please review again.

**Comment by cosmo0920** [comment]
:+1:

File: server/test/DBTablesTest.cc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2013-2014 Project Hatohol
+ * Copyright (C) 2013-2015 Project Hatohol
```
Comment (cosmo0920) [inline server/test/DBTablesTest.cc]: If this change including in this PR, please update copy right years in other headers and sources.

Comment (kz0817) [inline server/test/DBTablesTest.cc]: OK. I'll modify other source files.

Comment (cosmo0920) [inline server/test/DBTablesTest.cc]: Thanks!

Comment (kz0817) [inline server/test/DBTablesTest.cc]: I've done it. Please review again.

**Comment by cosmo0920** [comment]
@ashie Any objections? If nothing, I'll merge this PR.

**Comment by ashie** [comment]
No problem.

**Comment by kz0817** [comment]
おーきに。

**Pull request merged**